contracts/TellorFlex.sol

```diff
@@ -71,17 +73,32 @@ contract TellorFlex {
     constructor(
         address _token,
         address _governance,
+        address _owner,
         uint256 _stakeAmount,
         uint256 _reportingLock
     ) {
         require(_token != address(0), "must set token address");
         require(_governance != address(0), "must set governance address");
         token = IERC20(_token);
         governance = _governance;
+        owner = _owner;
         stakeAmount = _stakeAmount;
         reportingLock = _reportingLock;
     }
 
+    function init(address _governanceAddress) external {
+        require(msg.sender == owner);
+        require(initted == 0);
+        require(
+            _governanceAddress != address(0),
+            "governance address can't be zero address"
+        );
+
+        governance = _governanceAddress;
+        initted = 1;
```

Comment:
couldn't we remove `initted` and just `require(governance == address(0))` so that `init` can only be called once?

Reply:
and then we'd remove the `_governance` argument from the constructor

Reply:
good call, will do!